source/getting-started/index.markdown

```diff
@@ -41,11 +41,20 @@ Follow this guide if you want to get started with Home Assistant easily, or if y
 
 We will need a few things to get started with installing Home Assistant. The latest Raspberry Pi model makes a good and affordable starting point for your home automation journey. Links below are linking to Amazon US. If you're not in the US, you should be able to find these items in web stores in your country.
 
-- [Raspberry Pi 3 Model B+](https://amzn.to/2IAyNl0) + [Power Supply](https://www.raspberrypi.org/help/faqs/#powerReqs) (at least 2.5A)
+- [Raspberry Pi 4 Model B](https://www.raspberrypi.org/products/raspberry-pi-4-model-b/), 1 GB RAM version is enough, but if you can afford it, 2 or 4 GB RAM won't hurt + [Power Supply](https://www.raspberrypi.org/products/type-c-power-supply/) (at least 3.0A)
```

Comment:
Again, as with the previous PR, you've removed our links.

Reply:
I changed them for newest Pi version

And I asked for you to generate links, I can't generate them myself